src/components/job-row.tsx

```diff
@@ -9,6 +9,7 @@ import { useTranslator } from '../hooks';
 import { ICreateJobModel } from '../model';
 import DownloadIcon from '@mui/icons-material/Download';
 import StopIcon from '@mui/icons-material/Stop';
+import { Link } from '@mui/material';
 import { IconButton, Stack, TableCell, TableRow } from '@mui/material';
```

Comment:
```suggestion
import { IconButton, Link, Stack, TableCell, TableRow } from '@mui/material';
```

eslint may insist on restyling this further

Reply:
👍